src/graph-builders/graph-builders-registry.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2020 Obsidian Better Graph View
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Class for a graph builders registry.
```

Comment:
"Registry of graph builers. Allows us to easily add new graph builders in the future"